### DISCUSSION TITLE
Curator logs errors for most actions when it has nothing to do

### DISCUSSION THREAD

**Discussion by petiepooo** [Security-Onion-Solutions/securityonion#10496] [Unsupported Versions] [answered]
While debugging an issue with Curator not deleting for space (#10464), I noticed hundreds of lines like:

> 2023-06-01 14:10:38,175 ERROR     Unable to complete action "close".  No actionable items in list: <class 'curator.exceptions.NoIndices'>
2023-06-01 14:10:52,986 ERROR     Unable to complete action "close".  No actionable items in list: <class 'curator.exceptions.NoIndices'>

They were all the close and warm actions, whereas the delete actions all showed something like:

> 2023-06-01 14:10:06,118 INFO      Skipping action "delete_indices" due to empty list: <class 'curator.exceptions.NoIndices'>

Obviously, this is an expected outcome, and not a true error, but it appears the delete action uses `ignore_empty_list: True` whereas the close and warm actions use `continue_if_exception: False`.

Security Onion Solutions devs, is there a reason behind this?  If not, would it make sense to use `ignore_empty_list: True` on all actions?  In my testing, changing selected close actions to `ignore_empty_list: True` removes the error and I haven't noticed any ill effects.  Being able to search the log for 'ERROR' and only see actual errors would make troubleshooting easier.

**Comment by weslambert** [comment, marked as answer]
I've created an issue to make sure this is implemented where applicable in 2.4.

https://github.com/Security-Onion-Solutions/securityonion/issues/10512
